### PR TITLE
Ft layout recyclerview 155738443

### PR DESCRIPTION
**Pull request by meshnesh**
# What does this PR do?

It implements a RecyclerView which displays the user GitHub name and GitHub profile picture.

## Description of Task to be completed?

* Install `RecyclerView` library, `CircleImageView` and `Glide`.
* Change Main layout to `RelativeLayout`, create `layout_listitem` for displaying data.
* Create RecyclerView Adapter file.
* Created data objects to be used.
* Link adapter to the `Mainactivity`.

## How should this be manually tested?

* Clone the repo `git clone https://github.com/meshnesh/android-level-up`.
* Open the project on Android studio.
* Run the application either on an emulator device or android device connected to the laptop.

### What are the relevant pivotal tracker stories?

[#155738443](https://www.pivotaltracker.com/story/show/155738443)

### Screenshots

screenshot_20180626-141608

### CODE REVIEW

File: app/src/main/java/com/example/antonyng/level_up/MainActivity.java [hunk 1]
```diff
@@ -2,12 +2,59 @@
...
         setContentView(R.layout.activity_main);
+        Log.d(TAG, "onCreate: Started");
+
+        initImageBitmaps();
+    }
+
+    private void initImageBitmaps() {
+        Log.d(TAG, "initImageBitmaps: preparing  bitmaps");
+
+        mImageUrls.add("https://avatars2.githubusercontent.com/u/15978054?s=460&v=4");
```
Comment (Jak-Sidious) [inline app/src/main/java/com/example/antonyng/level_up/MainActivity.java:31]: I believe there is a more efficient way to add multiple elements to an Array list but this approach works well. However do look into another alternative to for your personal research.

Comment (meshnesh) [reply app/src/main/java/com/example/antonyng/level_up/MainActivity.java:31]: okay

**Comment by Jak-Sidious** [comment]
1. Does `CMD + B` work on all emulators to build the application or is this a personal shortcut and if the later, is it appropriate to add this to your PR documentation on **How Should this be Manually Tested**.

**Comment by Jak-Sidious** [comment]
Failing build is alright for circleci, You can implement that in the day2 tasks as per the PT board
LGTM
Proceed

**Pull request merged**